pages/General-Commands.md

```diff
@@ -56,17 +56,17 @@ ___
 Opens a web interface to edit permissions data. After changes are saved, a command will be given that you need to run for the changes to take effect.
 
 ___
-#### `/lp verbose <on|record|off|upload> [filter]`  
+#### `/lp verbose <on|record|off|upload|command> [filter|<me|player> <command>]`  
```

Comment:
This is a bit confusing imo, instead could we separate it so there are two sections:

`/lp verbose <on|record|off|upload> [filter]`
`/lp verbose command <me|player> <command>`

Reply:
Never received a notification for this...

How exactly would you want this... Just a completely separate section after the verbose one, or more having it right after the first header?

Like this?
```markdown
#### `/lp verbose <on|record|off|upload> [filter]`

blablabla

---
#### `/lp verbose command <me|player> <command>`

blablabla
```
...or like this?
```markdown
#### `/lp verbose <on|record|off|upload> [filter]`
#### `/lp verbose command <me|player> <command>`

blablabla
```

Reply:
Decided to make it a separate section